spec/02-integration/06-invalidations/02-core_entities_invalidations_spec.lua

```diff
@@ -182,6 +182,9 @@ for _, strategy in helpers.each_strategy() do
       end)
 
       it("on update", function()
+        -- router(traditional_compatible) incremental build use timestamp to check update
+        ngx.sleep(1)
```

Comment:
We can remove this line because now we do not depend on timestamp.

Reply:
Let's fix it.